### ISSUE TITLE
Keyboard key to open suggestions in base picker

### ISSUE THREAD

**Issue by JoshuaLamusga** [OfficeDev/office-ui-fabric-react#9592]
Follow-up to pull request #9547

## Actual behavior:
There is no way to open picker suggestions by keyboard without affecting the input in the textbox.

## Expected behavior:
There should be an easily-discoverable keyboard action that shows picker suggestions as its only side effect. Users that have closed picker suggestions currently type to open it again, which may be slightly inconvenient.

## Documentation describing expected behavior:
Every action must be accessible by keyboard for accessibility.

**Comment by KevinTCoughlin** [comment]
Makes sense. Do you plan to make a PR @JoshuaLamusga adding the keyboard command? cc: @joschect

**Comment by JoshuaLamusga** [comment]
I don't if someone else can take it. The issue existed before my PR, though it became easier to identify as a missing scenario after my PR went in. I'm trying to keep my focus in one (different) repo for speed.

**Comment by KevinTCoughlin** [comment]
> I don't if someone else can take it. The issue existed before my PR, though it became easier to identify as a missing scenario after my PR went in. I'm trying to keep my focus in one (different) repo for speed.

OK, I'm going to have the component owners backlog review this item as it will require discussion re: the interaction pattern and key to be used.

**Comment by joschect** [comment]
@JoshuaLamusga I'm a bit late with my response but my suggestion would be to use the floating picker. It gives you a lot more control over what you can do. The current picker is much more opinionated in the way that things should be done. Here is a [codesandbox](https://codesandbox.io/s/floating-picker-opens-with-key-press-idg4l) with a floating picker that opens on down arrow key pressed.

**Comment by khmakoto** [comment]
@JoshuaLamusga please let us know if the fix @joschect provided above is good enough for you or if there is anything else we can do to help!
